### PR TITLE
misc: add issue templates

### PR DESCRIPTION
**Pull request by drymarau**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    attributes:
+      label: 'Expected behavior'
+      description: 'A clear and concise description of what you expected to happen.'
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: 'Version'
+      description: 'What library version are you using?'
+      placeholder: '1.0.0'
```
Comment (tpettersen) [inline .github/ISSUE_TEMPLATE/bug_report.yml:27]: We could for the sake of ease for the user set this to the current released version of the SDK every time we push a new version

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      label: 'Describe the solution you would like'
+      description: 'A clear and concise description of what you want to happen.'
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: 'Describe alternatives you have considered'
+      description: 'A clear and concise description of any alternative solutions or features you have considered.'
+    validations:
+      required: true
```
Comment (tpettersen) [inline .github/ISSUE_TEMPLATE/feature_request.yml:22]: Does alternatives have to be required? If I come up with "I would like to have DTMF", there is not quite many alternatives that would be relevant

**Review by tpettersen — APPROVED** [review]
👍 I added a couple of suggestions

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (drymarau) [reply .github/ISSUE_TEMPLATE/bug_report.yml:27]: 👍

**Pull request merged**